src/msw/utilsgui.cpp

```diff
@@ -209,6 +209,32 @@ void wxDrawLine(HDC hdc, int x1, int y1, int x2, int y2)
     MoveToEx(hdc, x1, y1, NULL); LineTo((HDC) hdc, x2, y2);
 }
 
+// Faster version for drawing horizontal/vertical lines with solid color
+void wxDrawHVLine(HDC hdc, int x1, int y1, int x2, int y2, COLORREF color, int width)
+{
+    wxASSERT(x1 == x2 || y1 == y2);
+
+    int w1 = width / 2;
+    int w2 = width - w1;
+    RECT r;
+    if ( y1 == y2 )
+    {
+        if ( x1 == x2 )
+            return;
+        SetRect(&r, x1, y1 - w1, x2, y1 + w2);
```

Comment:
```suggestion
        ::SetRect(&r, x1, y1 - w1, x2, y1 + w2);
```
Just to make it clear that it's a Win32 API function.